### PR TITLE
feat: add winter theme 🎅

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Adds a new winter theme to CoW Swap:
- Requires `featureFlags.isChristmasEnabled`
- Adjust success sound
- Adds Snow
- Winter theme dark/light mode logo and background images (desktop/medium/small)

https://github.com/user-attachments/assets/524492df-9bdc-44b8-a7b8-aa259966154d

https://github.com/user-attachments/assets/ecc5d898-58a5-4e0b-84ca-206afbf1e65d

### CODE REVIEW

File: apps/cowswap-frontend/src/modules/application/containers/App/styled.ts [hunk 1]
```diff
@@ -44,6 +50,13 @@ export const BodyWrapper = styled.div<{ customTheme?: CowSwapTheme }>`
 
       if (customTheme === ('darkHalloween' as CowSwapTheme)) {
         backgroundImage = `url(${IMAGE_BACKGROUND_DARK_HALLOWEEN})`
+      } else if (
+        customTheme === ('darkChristmas' as CowSwapTheme) ||
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/modules/application/containers/App/styled.ts:54]: Nitpick: `as CowSwapTheme` is redundant here

Comment (shoom3301) [reply apps/cowswap-frontend/src/modules/application/containers/App/styled.ts:54]: Might be just: 
`['darkChristmas', 'lightChristmas'].includes(customTheme)`

**Review by shoom3301 — APPROVED** [review]
It looks amazing!

**Review by elena-zh — COMMENTED** [review]
Hey @fairlighteth , 
great job!

Some issues: 
1. on the 1st app opening, I can see this huge logo (both in a mobile device and in a desktop version)
image
2. On a Traded (success event) I hear a regular sound. Is this expected? 
Thanks!

**Review by anxolin — APPROVED** [review]

File: apps/cowswap-frontend/src/modules/application/containers/App/styled.ts (hunk 1 above)
Comment (anxolin) [reply apps/cowswap-frontend/src/modules/application/containers/App/styled.ts]: Refactored the christmas theme detection 57f809bafd5b9d2c939287e42a3d55bdd3ecaaa0

**Comment by anxolin** [comment]
> on the 1st app opening, I can see this huge logo (both in a mobile device and in a desktop version)

@elena-zh i can't reproduce that one. I tried in IOS device and incognito mode and doesn't happen to me. I even tried to throttle the internet speed, and it works fine to me

> On a Traded (success event) I hear a regular sound. Is this expected?

I added a new sound

**Comment by elena-zh** [comment]
> mode and doesn't happen to me. I even tried to throttle the internet speed, and it works fine to me
Cool then! Because it may happen to me in incognito mode when I open the app there for the 1st time (reproducible in Chrome browser). 
image


The sound is great, thank you for adding it!

**Pull request merged**